package.json

```diff
@@ -125,6 +125,7 @@
     "yup": "^0.32.0"
   },
   "resolutions": {
+    "minimist": "1.2.8",
```

Comment:
Instead of adding a resolution, we can simply update the minimist version in the dependencies above to `1.2.8`. Is there any issue with that? We only go for the resolutions when we face dependency conflicts and there is no other way but this one seems simpler and worked for me locally/